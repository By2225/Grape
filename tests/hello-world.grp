fun Int main() { print("Hello World!"); }
