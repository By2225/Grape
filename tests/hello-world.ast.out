
Int main()
{
prints(Hello World!);
}
